lib/config.js

```diff
@@ -25,6 +26,12 @@ if (argv.config && argv.config === true) {
     throw new Error('--config defined without any following configFile')
 }
 
+if (argv.saveTempFiles) {
+    logger.i('--saveTempFiles option: Guerilla temp files in the OS temp directory will not be deleted at process exit.');
```

Comment:
`saveTempFiles` is a slightly ambiguous name without the description. They are always saved. :smile: